virtcontainers/pkg/types/pcipath.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2020 Red Hat
```

Comment:
now the header is different

`Copyright (c) 2020 David Gibson, Red Hat Inc.`

Reply:
Uh... I'm confused, the one @devimc quotes should be the old version, not the new one.